WEEK 08/js/app.js

```diff
@@ -0,0 +1,92 @@
+var obj = {
+	data: {
+		user:'Yang',
+		message: 'Woot my first message!',
+	}
+}
+
+var url = 'https://fewd20.herokuapp.com/chat';
+
+var usedNames= [];
+
+$(document).ready(function(){
+	$.get(url, function(data, status){
+
+		for(var i=0; i< data.length; i++){
+
+			if (usedNames.indexOf(data[i].name) === -1){
+				usedNames.push(data[i].name);
+				$('#printname').append(data[i].user);
+				$('#printmessage').append(data[i].message);
+			}
+		}
+	}).fail(function(){
+
+	});
+})
+
+$('#send').click(function(){
```

Comment:
you should put your 

```
var obj = {
    data: {
        user: $('#user').val(),
        message: $('#message').val()
    }
}
```

So you can actually send the data that the user typed